### ISSUE TITLE
Copy and paste error

### ISSUE THREAD

**Issue by Lyroki17** [fonol/anki-search-inside-add-card#122]
When I copy and paste a long excerpt of the text on the field it messes up the spaces between words, commas and periods (even with high quality pdf files). Here is an example: 

"Connecting continuity with change , this edition features a new title that cements this work's past. Prior editions were entitled The Law and Practice of International Commercial Arbitration . This editionis entitled Redfern and Hunter on International Arbitration . Thus, we have formally adopted the title by which this work is in any event universally . "

![Add-on error](https://user-images.githubusercontent.com/50713804/86181738-fac99680-bb04-11ea-8ba3-d0d772063b6c.png)

Thank you so much for the attention and keep up with the awesome work.

**Comment by fonol** [comment]
I also have issues with copying text, but the ones in your screenshot, I see for the first time.
Are your most common issues across different pdfs like that, i.e. extra spaces between punctuation symbols?
Because I haven't come across a pdf where it does that, for me, the most common problem are that it won't copy a space between the last word on a line and the first word on the next line.

**Comment by Lyroki17** [comment]
Thank you for the fast reply! 
The most common issue related to copying text is these spaces between ponctuation symbols and spaces inside the word (_i.e._ Sear ching, PDF readi ng). Sometimes it cuts the space between two different words also even words that are in the same line.

**Comment by fonol** [comment]
Did the update from like 2 days ago improve things for you?

**Comment by Lyroki17** [comment]
Indeed, it improved! However the gap between words is still happening. Thanks!
